manim/mobject/types/vectorized_mobject.py

```diff
@@ -182,7 +184,44 @@ def update_rgbas_array(self, array_name, color=None, opacity=None):
             curr_rgbas[:, 3] = rgbas[:, 3]
         return self
 
-    def set_fill(self, color=None, opacity=None, family=True):
+    def set_fill(self, color=None, opacity=None, family=True) -> "VMobject":
+        """Set the fill color and fill opacity of a :class:`VMobject`
+        including its subMobjects.
```

Comment:
The second part about the submobjects depends on the ``family`` parameter and should be mentioned separately.

```suggestion
        """Set the fill color and fill opacity of a :class:`VMobject`.
```